spec/integration/import_rocket_fuel_data_into_postgres_spec.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'rocketfuel_api'
+
+RSpec.describe 'Import Rocketfuel data into Postgres', :vcr, :db do
+  subject(:etl) do
+    Importeur::ETL.new(
+      extractor: Importeur::Extractor.new(
+        rocketfuel_data_source,
+        cursor,
+        'rocketfuel-advertisers'
+      ),
+      transformer: transformer,
+      loader: Importeur::ActiveRecordPostgresLoader.new(Affiliate, :id)
+    )
+  end
+
+  let(:rocketfuel_data_source) do
+    Importeur::DataSources::Rocketfuel.new(rocketfuel_service)
+  end
+
+  let(:rocketfuel_service) do
+    RocketfuelApi::Service::Company.new(rocketfuel_connection)
+  end
+
+  let(:rocketfuel_connection) do
+    # TODO: Change to production API endpoint
```

Comment:
What's wrong with using sandbox api for testing?

Reply:
Good question, just thought would be good to take the real endpoint. But in the end it doesnt matter i guess.